custom_components/tuya_local/devices/fresco_hydrateultra_petfountain_alt.yaml

```diff
@@ -0,0 +1,233 @@
+name: Pet fountain
+products:
+  - id: avhcsv4u4nj05otr
+    name: Petoneer Fresco Hydrate Ultra alternative
+
+primary_entity:
+  entity: switch
+  icon: "mdi:water-pump"
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+
+secondary_entities:
+  - entity: sensor
+    name: Water level
+    icon: "mdi:water"
+    category: diagnostic
+    dps:
+      - id: 9
+        type: string
+        unit: "%"
+        name: sensor
+        mapping:
+          - dps_val: "1"
+            value: 25
+          - dps_val: "2"
+            value: 50
+          - dps_val: "3"
+            value: 75
+          - dps_val: "4"
+            value: 100
+
+  - entity: light
+    translation_key: indicator
+    category: config
+    dps:
+      - id: 12
+        name: brightness
+        type: integer
+        range:
+          min: 1
+          max: 10
+
+  - entity: sensor
+    name: Water quality
+    icon: "mdi:water"
+    category: diagnostic
+    dps:
+      - id: 13
+        type: integer
+        name: sensor
+        unit: TDS
+
+  - entity: button
+    name: UV sterilization
+    icon: "mdi:lightbulb-fluorescent-tube"
+    category: config
+    dps:
+      - id: 14
+        type: boolean
+        name: button
+        optional: true
+
+  - entity: sensor
+    name: UV runtime
+    class: duration
+    category: diagnostic
+    dps:
+      - id: 15
+        type: integer
+        name: sensor
+        unit: s
+
+  - entity: sensor
+    name: Last Water change
+    icon: "mdi:water-circle"
+    category: diagnostic
+    class: duration
+    dps:
+      - id: 101
+        type: integer
+        name: sensor
+        range:
+          min: 0
+          max: 7200
+        unit: min
+
+  - entity: sensor
+    name: Water health
+    icon: "mdi:water-circle"
+    category: diagnostic
+    class: duration
+    dps:
+      - id: 101
+        type: integer
+        name: sensor
+        range:
+          min: 0
+          max: 7200
+        mapping:
+          - target_range:
+              min: 100
+              max: 0
+        precision: 1
+        unit: "%"
+
+  - entity: button
+    name: Reset Water health
+    icon: "mdi:sync"
+    category: config
+    dps:
+      - id: 101
+        type: integer
+        name: button
+        mapping:
+          - dps_val: 0
+            value: true
+
+  - entity: sensor
+    name: Last Filter change
```

Comment:
As with above, it might be better to invert the value so it is a countdown and rename to "Filter change due" or "Filter remaining"

The unit should be `min` not `m` (meters)

Reply:
inverted the value and fixed unit.